style.css

```diff
@@ -0,0 +1,181 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap");
+
+:root {
+  --color-white: #f3f3f3;
+  --color-darkblue: #1b1b32;
+  --color-darkblue-alpha: rgba(27, 27, 50, 0.8);
+  --color-green: #37af65;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: var(--color-white);
+  margin: 0;
+}
+
+/* mobile friendly alternative to using background-attachment: fixed */
```

Comment:
it will be better if you removed comments, so your code will be cleaner.

Reply:
[OPTIONAL] It should be "linter<strike>**s**</strike> errors <strike>**with**</strike> **related to** spacing"
[OPTIONAL] "committing to linters" sounds like "changing linters.yml file" to me, which is forbidden. You probably meant "fixing linter errors".